CMAQ.md

```diff
@@ -33,313 +32,243 @@ Brief description: a three-dimensional Euler grid-based atmospheric chemistry an
 
 ### Installing dependencies
 
-```shell
+
 yum install time -y
 yum install curl* -y
-yum install wget -y
 yum install csh -y
 yum install zlib* -y
-```
 
 
 ### Installing GNU 9.3
 
-```shell
+
 yum install -y centos-release-scl
 yum install -y devtoolset-9-gcc
 yum install -y devtoolset-9-gcc-c++
+yum install -y devtoolset-9-gcc-gfortran
 yum install -y devtoolset-9-binutils
 scl enable devtoolset-9 bash
 echo "source /opt/rh/devtoolset-9/enable" >> /etc/profile
-```
+
 
 ### Installing Open MPI
 
 1. Install the system dependency package:
 
-```shell
+
 yum install libxml2* systemd-devel.aarch64 numa* -y
-```
+
 
 2. Install Open MPI:
 
-```shell
+
 wget https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.3.tar.gz
 tar -zxvf openmpi-4.0.3.tar.gz
 cd openmpi-4.0.3
-./configure --prefix=/path/to/OPENMPI --enable-pretty-print-stacktrace --enable-orterun-prefix-by-default  --with-cma --enable-mpi1-compatibility --enable-mpi-fortran=yes
-make -j $(nproc) all
```

Comment:
The removal of `all` target leads to https://github.com/Yikun/hpc-arm/pull/1#discussion_r686779754